cpp/include/Ice/Properties.h

```diff
@@ -43,126 +45,124 @@ namespace Ice
-        /// properties are returned.
+        /// Gets all properties whose keys begins with @p prefix. If @p prefix is an empty string, then all properties
+        /// are returned.
         /// @param prefix The prefix to search for (empty string if none).
         /// @return The matching property set.
         PropertyDict getPropertiesForPrefix(std::string_view prefix);
 
-        /// Set a property. To unset a property, set it to the empty string.
+        /// Sets a property. To unset a property, set it to the empty string.
         /// @param key The property key.
         /// @param value The property value.
         /// @see #getProperty
         void setProperty(std::string_view key, std::string_view value);
 
-        /// Get a sequence of command-line options that is equivalent to this property set. Each element of the returned
-        /// sequence is a command-line option of the form <code>--<em>key</em>=<em>value</em></code>.
+        /// Gets a sequence of command-line options that is equivalent to this property set. Each element of the
+        /// returned sequence is a command-line option of the form `--key=value`.
         /// @return The command line options for this property set.
         StringSeq getCommandLineOptions();
 
-        /// Convert a sequence of command-line options into properties. All options that begin with
-        /// <code>--<em>prefix</em>.</code> are converted into properties. If the prefix is empty, all options that
-        /// begin with <code>--</code> are converted to properties.
-        /// @param prefix The property prefix, or an empty string to convert all options starting with <code>--</code>.
+        /// Converts a sequence of command-line options into properties. All options that begin with `--prefix.` are
+        /// converted into properties. If the prefix is empty, all options that begin with `--` are converted to
+        /// properties.
+        /// @param prefix The property prefix, or an empty string to convert all options starting with `--`.
         /// @param options The command-line options.
         /// @return The command-line options that do not start with the specified prefix, in their original order.
         StringSeq parseCommandLineOptions(std::string_view prefix, const StringSeq& options);
 
-        /// Convert a sequence of command-line options into properties. All options that begin with one of the following
-        /// prefixes are converted into properties: <code>--Ice</code>, <code>--IceBox</code>, <code>--IceGrid</code>,
-        /// <code>--IceSSL</code>, <code>--IceStorm</code>, and <code>--Glacier2</code>.
+        /// Converts a sequence of command-line options into properties. All options that begin with one of the
+        /// following prefixes are converted into properties: `--Ice`, `--IceBox`, `--IceGrid`, `--IceSSL`,
+        /// `--IceStorm`, and `--Glacier2`.
```

Comment:
"All options that begin with one of the" is not true as it depends on the configured opt in prefixes.

I would remove the `and` and use `, etc.` as it's not complete. Or maybe just say something like:

```
Converts a sequence of command-line options that start with an Ice property prefix (`--Ice.`, `--IceSSL.`, etc.) into properties. 
```

@pepone  is correct, `--DataStorm` could be added to this list. 

This function still ultimately calls `setProperty` which checks the opt-in prefixes.

Reply:
Is there a user case for application code to call this? maybe we should we mark as internal to remove it from doxygen docs.

Reply:
Update to "etc". Other concerns are for a separate PR.